internal/deploy/cmd_deploy.go

```diff
@@ -13,6 +17,10 @@ import (
 	"github.com/altipla-consulting/wave/internal/query"
 )
 
+const (
+	MaxDeployAttempts = 2
```

Comment:
nit: Para no exportarla del paquete:

```suggestion
	maxDeployAttempts = 2
```